source/content/debian-php-appserver.md

```diff
@@ -0,0 +1,122 @@
+product: [--]
+integration: [--]
+---
+
+...
+
+
+## Overview
+
+Pantheon is introducing a new PHP application server based on the Debian operating system, which will replace our previous CentOS-based infrastructure. This upgrade represents our commitment to providing a modern, secure, and efficient hosting environment for your applications.
+
+Depending on your application's configuration, this change may have major effects on the compatibility of your site. During the beta phase, we invite you to test your site for compatibility and performance to ensure a seamless upgrade.
+
+## Why We're Making This Change
+
+- Enhanced security features with a hardened operating system image
+- Long Term Support (LTS) coverage for EOL PHP versions
+- Better compatibility with updated PHP extensions
+
+## Opt-In Beta Phase (Q2 2025)
+
+During the beta phase, you can opt in to test your sites on the new Debian-based infrastructure. The opt-in is environment-based, meaning you can trial Debian with a clone of your sites without any harm to your live environments. This allows you to:
+
+- Verify your site's compatibility before the platform-wide rollout
+- Test your deployment workflows
+- Identify any potential issues early
+- Provide valuable feedback to our team
+
+### How to Opt In
+
+To enable the Debian operating system for an environment, add the following to your `pantheon.yml`:
+
+```yaml:title=pantheon.yml
+appserver_os: debian12
+```
+
+To safely revert to the CentOS operating system, you may remove the line during the beta phase or use:
+
+```yaml:title=pantheon.yml
+appserver_os: centos
+```
+
+Since any `pantheon.yml` changes are part of your site repository and promoted if you push to test or live, we recommend using a multidev or removing the `appserver_os` parameter before promoting any code while you are testing compatibility.
+
+## Timeline
+
+| Date | Phase | Details |
+|-----------|------------------|--------------|
+| April - July 2025 | Beta (Opt-in) | Sites can be opted-in to Debian. All other sites will continue using CentOS |
+| August-September 2025 | Rollout (Opt-out) | A slow rollout will auto-upgrade all sites to Debian. Opting out via the `pantheon.yml` parameter will be available |
+| Q1 2026 | CentOS Deprecation | CentOS will no longer be a valid `appserver_os` option, and all remaining sites will be auto-upgraded to Debian |
+
+
+## Known Changes and Differences
+
+### OS Packages
+
+| Package | CentOS | Debian |
+|---------|---------|---------|
+| OIC (Oracle Instant Client) | Available for PHP 7.4-8.3 | Available for PHP 8.2-8.4 |
+| IBM PHP ODBC Driver | Available for PHP 7.4-8.2 | Not available |
+| wkhtmltopdf | 0.12.5 available | Not available |
+| Apache Tika | 1.18, v1.21 available | Not available* |
+| PhantomJS | 1.7.0, 2.1.1 available | Not available |
+
+* For `search_api_attachments` users, we recommend using the Solr Extractor built into the `Pantheon Search` Solr server, rather than using Tika on the appserver.
```

Comment:
Can we link to more info?

Reply:
https://www.drupal.org/docs/8/modules/file-extractor/search-api-solr-extractor ?

Reply:
Added the path to the configuration